src/containers/AccountContainer.js

```diff
@@ -44,17 +45,27 @@ class AccountContainer extends Component {
         }
       ]
     }
+    this.handleChange = this.handleChange.bind(this)
   }
 
   handleChange(event) {
-    // your code here
+    this.setState({
+      searchTerm: event.target.value
+    })
   }
 
-  render() {
+  componentDidMount(){
+    TransactionsAdapter.all()
+      .then(transactions => this.setState({
+          transactions: transactions
```

Comment:
you can use ES6 syntax if you like `{dog}` is the same as `{dog: dog}`. I don't usually use it, but your choice.